src/java/io/bazel/rulesscala/scalac/ScalacProcessor.java

```diff
@@ -0,0 +1,268 @@
+package io.bazel.rulesscala.scalac;
+
+import com.google.devtools.build.lib.worker.WorkerProtocol.WorkRequest;
+import com.google.devtools.build.lib.worker.WorkerProtocol.WorkResponse;
+import io.bazel.rulesscala.jar.JarCreator;
+import io.bazel.rulesscala.worker.GenericWorker;
+import io.bazel.rulesscala.worker.Processor;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.lang.reflect.Field;
+import java.nio.file.FileSystems;
+import java.nio.file.FileVisitResult;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.SimpleFileVisitor;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Enumeration;
+import java.util.List;
+import java.util.Map.Entry;
+import java.util.Map;
+import java.util.jar.JarEntry;
+import java.util.jar.JarFile;
+import scala.tools.nsc.*;
+import scala.tools.nsc.reporters.ConsoleReporter;
+
+class ScalacProcessor implements Processor {
+  /**
+   * This is the reporter field for scalac, which we want to access
+   */
+  private static Field reporterField;
+  static {
+    try {
+      reporterField = Driver.class.getDeclaredField("reporter"); //NoSuchFieldException
+      reporterField.setAccessible(true);
+    }
+    catch (Exception ex) {
+      throw new RuntimeException("nope", ex);
```

Comment:
I know this didn't change much before but I wonder if there isn't a better error message that can be reported